src/RiotAPILibraries.ts

```diff
@@ -217,34 +211,19 @@ export default class RiotAPILibraries {
         const editMessagePromise = message.channel.send(`Fetching the list of libraries for ${language}, this post will be edited with the result.`);
 
         let libraryDescriptions: LibraryDescription[] = [];
-        try {
-            let requiredTags;
-            if(message.channel.type == 'GUILD_TEXT') { // if this is the server text channel
-                requiredTags = this.settings.riotApiLibraries.requiredTagContextMap[message.channel.name]; //get the reqiured tags from settings.
-                if (requiredTags == undefined) {
-                    message.channel.send(this.settings.riotApiLibraries.wrongChannel
-                                         .replace('{channel}', message.channel.name)
-                                         .replace('{valid-channels}', 
-                                                  Array.from(this.settings.riotApiLibraries.requiredTagContextMap.keys())
-                                                  .map(e => '#' + e)
-                                                  .join(", ")
-                                                 )
-                                        );
-                    return;
-            } else {
-                requiredTags = this.allTagOptions; //in a dm report all libs
-            }
-            libraryDescriptions = (await this.getLibrariesForLanguage(language, requiredTags))
-                .sort((a, b) => b.stars - a.stars); // Sort by stars
-        }
-        catch (e) {
-            message.channel.send(e.message);
-            return;
+        let requiredTags = this.allTagOptions;
+        
+        if(message.channel.type == 'GUILD_TEXT') { // if this is the server text channel
```

Comment:
error TS2367: This condition will always return 'false' since the types '"text" | "dm" | "news"' and '"GUILD_TEXT"' have no overlap.

Reply:
fixed from "GUILD_TEXT" to "text", which should be the right keyword for discord.js..... though that's definitely not what the docs suggest:
Message.channel is of type [TextBasedChannels](https://discord.js.org/#/docs/discord.js/main/typedef/TextBasedChannels) which eventually as a aggregate type has ChannelType which contains a huge enum. None of this is a string union type but..... I guess I'll listen to the linter and not the docs?